server/api.js

```diff
@@ -134,4 +135,15 @@ router.post("/publish-tenders", (req, res) => {
 	res.status(200).json({ message: "Form submitted successfully!" });
 });
 
+router.get("/buyerTender/:id", async (req, res) => {
+	const buyerId = req.params.id;
```

Comment:
buyerId shouldn't come from parameters, it should come from the session. Since the sign in part is not completed yet, you can just hard-code any ID for now